tikv-configuration-file.md

```diff
@@ -18,6 +18,50 @@ This document only describes the parameters that are not included in command-lin
 + Default value: `1`
 + Minimum value: `1`
 
+<<<<<<< HEAD
```

Comment:
@breeswish hi please help confirm if the changes apply to 3.0 version, thanks~

Reply:
Hi, this change is included in 3.0 version. Thanks!

Reply:
Hi @breeswish @andylokandy Could you please help resolve the conflicts then? I'm not sure about the final version that is technically right.

And "However the doc should be updated after corresponding new version is release" means this PR should not be merged until the new 3.0 version is released?